src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const array = [...document.querySelectorAll('li')];
+
+const toNum = (el) => {
+  return parseFloat(el.replace(/[$,]/g, ''));
+};
+
+const sortList = (list) => {
+  list.sort((a1, a2) => toNum(a2.dataset.salary) - toNum(a1.dataset.salary));
+
+  list.forEach((element) => {
+    document.querySelector('ul').insertAdjacentElement('beforeend', element);
```

Comment:
This line moves each element to the end of the list, effectively sorting the list in the DOM. Ensure that the 'ul' element is the correct parent for these 'li' elements.